ComposeAdvanced/app/src/main/java/com/example/android/wearable/composeadvanced/presentation/ui/landing/LandingScreen.kt

```diff
@@ -67,68 +67,74 @@ fun LandingScreen(
     Box(modifier = Modifier.fillMaxSize()) {
 
         // Places both Chips (button and toggle) in the middle of the screen.
-        Column(
-            modifier = Modifier
-                .fillMaxSize()
-                .padding(horizontal = 10.dp),
-            verticalArrangement = Arrangement.spacedBy(4.dp, Alignment.CenterVertically)
+        ScalingLazyColumn(
+            modifier = Modifier.fillMaxSize(),
+            state = ScalingLazyListState(initialCenterItemIndex = 1)
         ) {
-            CompactChip(
-                onClick = onClickWatchList,
-                label = {
-                    Text(
-                        stringResource(R.string.list_of_watches_button_label),
-                        maxLines = 1,
-                        overflow = TextOverflow.Ellipsis
-                    )
-                }
-            )
+            item {
+                // Signify we have drawn the content of the first screen
+                ReportFullyDrawn()
```

Comment:
Does this work - do we need it? I am confused as to why this is embedded inside one of the items - should it be a launched effect?

Reply:
Yep - it's copied from the performance samples.  

https://github.com/android/performance-samples/blob/357887bc8b523ec06d3225ea9920eb6374ac66da/MacrobenchmarkSample/app/src/main/java/com/example/macrobenchmark/target/FullyDrawnStartupActivity.kt

It's there so it's in the frame when the ScalingLazyColumn content is drawn. Not above the ScalingLazyColumn, where it's triggered before a paint with real content.